src/ui/components/ObjectView.vue

```diff
@@ -6,11 +6,16 @@
 import _ from "lodash";
 import StyleRuleManager from "@/plugins/condition/StyleRuleManager";
 import {STYLE_CONSTANTS} from "@/plugins/condition/utils/constants";
+const defaultOptions = {
+    layoutFontSize: '',
+    layoutFont: '',
+    showEditView: false,
```

Comment:
The `show` function already has an argument that tells the view whether to render in edit mode or not. Adding another flag to override that is just confusing.

Reply:
OK. This already existed as props before this PR, but I can fix this.

Reply:
Removed from options and returned to how it was before.